watertap/unit_models/cstr.py

```diff
@@ -0,0 +1,84 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+CSTR unit model for BSM2 and plant-wide wastewater treatment modeling.
+This unit inherits from the IDAES CSTR unit.
+"""
+
+# Import IDAES cores
+from idaes.core import (
+    declare_process_block_class,
+)
+from idaes.models.unit_models.cstr import CSTRData
+
+import idaes.logger as idaeslog
+
+from pyomo.environ import (
+    Constraint,
+    NonNegativeReals,
+    Var,
+    units as pyunits,
+)
+
+from watertap.costing.unit_models.cstr import cost_cstr
+
+__author__ = "Marcus Holly"
+
+
+# Set up logger
+_log = idaeslog.getLogger(__name__)
+
+
+@declare_process_block_class("CSTR")
```

Comment:
Renaming this from AnoxicCSTR to CSTR is causing a pylint issue. Is there a way to get around this and, if not, what should the new name be?

Reply:
I'm not 100% sure, but is it at all possible that the thing Pylint is complaining about is that a derived class is being defined with the same name as its base class? Since `CSTRData` was imported, defining a new `CSTRData` class would result in a different object (i.e. the derived `CSTRData`) reusing the same name as another object (i.e. the `CSTRData` imported from `idaes.models.unit_models.cstr`).

One way to check this could be to change the import alias, e.g. `from idaes.models.unit_models.cstr import CSTRData as CSTRDataFromIDAES` (or something).

Reply:
@lbianchi-lbl I believe you're right - I just didn't explain it well enough in my initial comment. I'll incorporate your change and see if it resolves the issue. Thanks!